src/gantt.ts

```diff
@@ -1892,48 +1892,57 @@ export class Gantt implements IVisual {
+                }
+
+                // add current task
+                const taskRecord = <GroupedTask>{
+                    name,
+                    tasks: groupedTasks[key]
+                };
+                result.push(taskRecord);
+                alreadyReviewedKeys.push(key);
+
+                // see all the children and add them
+
+                for (const task of groupedTasks[key]) {
+                    if (task.children && !collapsedTasks.includes(task.name)) {
+                        for (const childrenTask of task.children) {
+                            const childrenFullName = `${name}.${childrenTask.name}`;
+                            const isChildrenKeyAlreadyReviewed = alreadyReviewedKeys.includes(childrenFullName);
+
+                            if (!isChildrenKeyAlreadyReviewed) {
+                                const childrenRecord = <GroupedTask>{
+                                    name: childrenTask.name,
+                                    tasks: groupedTasks[childrenFullName]
+                                };
+                                result.push(childrenRecord);
+                                alreadyReviewedKeys.push(childrenFullName);
+                            }
 
-                    // add current task
-                    const taskRecord = <GroupedTask>{
-                        name,
-                        tasks: groupedTasks[key]
-                    };
-                    result.push(taskRecord);
-                    alreadyReviewedKeys.push(key);
-
-                    // see all the children and add them
-                    groupedTasks[key].forEach((task: Task) => {
-                        if (task.children && !collapsedTasks.includes(task.name)) {
-                            task.children.forEach((childrenTask: Task) => {
-                                const childrenFullName = `${name}.${childrenTask.name}`;
-                                const isChildrenKeyAlreadyReviewed = alreadyReviewedKeys.includes(childrenFullName);
-
-                                if (!isChildrenKeyAlreadyReviewed) {
-                                    const childrenRecord = <GroupedTask>{
-                                        name: childrenTask.name,
-                                        tasks: groupedTasks[childrenFullName]
-                                    };
-                                    result.push(childrenRecord);
-                                    alreadyReviewedKeys.push(childrenFullName);
-                                }
-                            });
                         }
-                    });
+                    }
+
                 }
-            });
+
+            }
 
             result.forEach((x, i) => {
                 x.tasks.forEach(t => t.index = i);
                 x.index = i;
             });
 
+            // Ensure that the tasks with children are displayed first
```

Comment:
https://dev.azure.com/powerbi/PowerBICustomVisuals/_workitems/edit/1421305

add this url for the ancestors